src/client/renderer/r_particle.c

```diff
@@ -293,12 +312,22 @@ void R_DrawParticles(const r_element_t *e, const size_t count) {
 	R_BindArray(R_ARRAY_ELEMENTS, &r_particle_state.element_buffer);
 
 	const GLuint base = (uintptr_t) e->data;
+	r_particle_type_t last_type = -1;
+	GLuint last_texnum = -1;
```

Comment:
Type 0 is a valid type, same same with last_texnum (particle with no image may be valid, like coronas)